src/index.ts

```diff
@@ -25,8 +26,17 @@ export const options: Record<
         type: 'path',
         category: 'Global',
         array: true,
-        default: [{ value: [] }],
-        description: 'Provide an order to sort imports.',
+        default: [
+            {
+                value: [
+                    // node.js built-ins are always first
+                    THIRD_PARTY_MODULES_SPECIAL_WORD, // Everything not matching relative imports
+                    '^[.]', // relative imports
```

Comment:
Should this be `^[./]`?  That seems to be what we use in our tests, at least...

Reply:
I don’t think so?

- `^[./]` matches `/foo/bar`, `../foo/bar` , `./foo/bar` 
- `^[.] only matches `../foo/bar`, `./foo/bar` — and doesn’t match file-system absolute-path imports

This is exactly the confusion I was a little worried about.

Reply:
I don’t think file-system absolute-path imports are that common, but yeah

Reply:
[I see now that you edited your github comment after I had loaded the page, but before I replied, lol]

Reply:
Alright, let's leave it the way you've got it.  Absolute file system imports would be weird, and should probably deserve an explicit importOrder.